pages/shoppingcart.js

```diff
@@ -33,3 +43,5 @@ function ShoppingCart() {
 }
 
 export default ShoppingCart;
+
+const RemoveButton = styled.button``;
```

Comment:
you should remove this line, as you are not using the component and it is empty on top of that.
```suggestion
```